assets/js/buttons-examples.js

```diff
@@ -235,6 +235,12 @@ $(document).ready(function () {
       ],
       children: [],
     },
+    {
```

Comment:
Please add this button type in the correct alphabetical order (at the time of writing this, under Animated)

Reply:
Yup I'll do that just now - sorry for missing that out!